### PR TITLE
Adding tag heartbeat duration configuration option.

### PR DESCRIPTION
**Pull request by phil-impinj**

### CODE REVIEW

File: src/main/java/com/impinj/itemsense/client/coordinator/recipe/Recipe.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
     private RecipeType type;
     private String readerConfigurationName;
     private Integer tagHeartbeatMinutes;
```
Comment (davidyscott) [inline src/main/java/com/impinj/itemsense/client/coordinator/recipe/Recipe.java]: Does it make sense to deprecate this now?  to discourage people from coding against it, and then we could remove it from a hypothetical 2017r2 release?

Comment (natke) [inline src/main/java/com/impinj/itemsense/client/coordinator/recipe/Recipe.java]: Yes, I think this does make sense

**Comment by phil-impinj** [comment]
@natke - field deprecated and test added.

**Comment by phil-impinj** [comment]
@natke - specific test around prior serialized version added.

**Pull request merged**